frontend/src/components/AddActionItems.js

```diff
@@ -0,0 +1,191 @@
+import React, { useState } from "react";
+import { useSelector } from "react-redux";
+
+import GetActionItems from "./GetActionItems";
+import NextButton from "./NextButton";
+
+import { ACTION_PLAN_URL, ACTIVE_URL } from "../utils/constants";
+
+import styled from "styled-components";
+
+const AddActionItems = () => {
+  const [newAction, setNewAction] = useState("");
+  const [name, setName] = useState("");
+
+  const retroId = useSelector((store) => store.retro);
+
+  //Patch to send the retro to inactive
+  const handleActiveRetro = (event) => {
+    event.preventDefault();
+
+    const options = {
+      method: "PATCH",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        active: false,
+      }),
+    };
+    fetch(ACTIVE_URL(`${retroId._id}`), options)
+      .then((res) => res.json())
+      .then((data) => data);
+  };
+
+  const RefreshButton = () => {
+    window.location.reload("Refresh");
+  };
```

Comment:
Is it really necessary to refresh whole page? Wouldn't making GET request and updating redux store be enough?
Moreover, it should be `refreshButton`, not `RefreshButton`